optaplanner-spring-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/OptaPlannerAutoConfiguration.java

```diff
@@ -40,6 +40,10 @@
 import org.optaplanner.core.config.solver.SolverManagerConfig;
 import org.optaplanner.core.config.solver.termination.TerminationConfig;
 import org.optaplanner.persistence.jackson.api.OptaPlannerJacksonModule;
+import org.optaplanner.spring.boot.autoconfigure.config.OptaPlannerProperties;
```

Comment:
Note: the reason why imports are introduced yet no other changes is because the old classes lived in `org.optaplanner.spring.boot.autoconfigure` (same package as this class), and the new classes live in `org.optaplanner.spring.boot.autoconfigure.config`